src/librustc_errors/emitter.rs

```diff
@@ -78,6 +83,28 @@ impl Emitter for EmitterWriter {
                                    &children,
                                    &suggestions);
     }
+
+    fn show_explain(&mut self) -> Vec<String> {
+        if !self.short_message && !self.error_codes.is_empty() {
+            let mut error_codes = self.error_codes.clone().into_iter().collect::<Vec<_>>();
+            error_codes.sort();
+            if error_codes.len() > 1 {
+                let limit = if error_codes.len() > 9 { 9 } else { error_codes.len() };
+                vec![format!("You've got a few errors: {}{}",
+                             error_codes[..limit].join(", "),
+                             if error_codes.len() > 9 { "..." } else { "" }),
+                     format!("If you want more information on an error, try using \
+                              \"rustc --explain {}\"",
```

Comment:
(If we end up needing to regenerate the UI test expectations anyway in the course of this PR, then we might as well also update the language (and use backticks) at the same time; but if not, then we can continue leave those tasks to #48559.)